yt_dlp/extractor/cybrary.py

```diff
@@ -0,0 +1,147 @@
+            'uploader': 'Cybrary',
+            'chapter': 'Implement Network Security',
+            'chapter_id': '52693'
+        },
+        'expected_warnings': ['No authenticators for vimeo']
+    }]
+
+    def _real_extract(self, url):
+        activity_id, enrollment_id = self._match_valid_url(url).group('id', 'enrollment')
+        course = self._call_api('enrollment', enrollment_id)['content']
+        activity = next((activity for activity in (traverse_obj(course, ('learning_modules', ..., 'activities', ...)) or [])
+                        if int(activity_id) == activity.get('id')), None)
+
+        if activity.get('type') not in ['Video Activity', 'Lesson Activity']:
+            raise ExtractorError('The activity is not a video')
+
+        module = next((m for m in traverse_obj(course, ('learning_modules')) or []
+                      if int(activity_id) in traverse_obj(m, ('activities', ..., 'id') or [])), None)
+
+        vimeo_id = self._get_vimeo_id(activity_id)
+
+        return {
+            '_type': 'url_transparent',
+            'series': traverse_obj(course, ('content_description', 'title')),
+            'series_id': str_or_none(traverse_obj(course, ('content_description', 'id'))),
+            'id': vimeo_id,
+            'chapter': module.get('title'),
+            'chapter_id': str_or_none(module.get('id')),
+            'title': activity.get('title'),
+            'url': smuggle_url(f'https://player.vimeo.com/video/{vimeo_id}', {'http_headers': {'Referer': 'https://api.cybrary.it'}})
+        }
+
+
+class CybraryCourseIE(CybraryBaseIE):
+    _VALID_URL = r'https://app.cybrary.it/browse/course/(?P<id>[\w-]+)/?(?:$|[#?])'
+    _TESTS = [{
+        'url': 'https://app.cybrary.it/browse/course/az-500-microsoft-azure-security-technologies',
+        'info_dict': {
+            'id': 898,
+            'title': 'AZ-500: Microsoft Azure Security Technologies',
+            'description': 'md5:69549d379c0fc1dec92926d4e8b6fbd4'
+        },
+        'playlist_count': 59
+    }, {
+        'url': 'https://app.cybrary.it/browse/course/cybrary-orientation',
+        'info_dict': {
+            'id': 1245,
+            'title': 'Cybrary Orientation',
+            'description': 'md5:9e69ff66b32fe78744e0ad4babe2e88e'
+        },
+        'playlist_count': 4
+    }]
+
+    def _real_extract(self, url):
+        course_id = self._match_id(url)
+        course = self._call_api('course', course_id)
+        enrollment_info = self._call_api('course_enrollment', course['id'])
+
+        entries = [self.url_result(
+            f'https://app.cybrary.it/immersive/{enrollment_info["id"]}/activity/{activity["id"]}')
+            for activity in traverse_obj(course, ('content_item', 'learning_modules', ..., 'activities', ...))]
+
+        return self.playlist_result(
+            entries, traverse_obj(course, ('content_item', 'id')), course.get('title'), course.get('short_description'))
```

Comment:
```suggestion
        return self.playlist_result(
            entries,
            traverse_obj(course, ('content_item', 'id'), expected_type=str_or_none),
            course.get('title'), course.get('short_description'))
```
id shouldn't be a number